### PR TITLE
Second general clean-up

### PR DESCRIPTION
**Pull request by sjaeckel**
* Use the same header/footer for all files
* Fix some open issues

### CODE REVIEW

**Comment by karel-m** [comment]
There are still some old URLs 
```
$ find src tests demos -type f | xargs grep libtom.org
tests/no_prng.c: * http://libtom.org
demos/constants.c: * Tom St Denis, tomstdenis@gmail.com, http://libtom.org
demos/sizes.c: * Tom St Denis, tomstdenis@gmail.com, http://libtom.org
```

**Comment by karel-m** [comment]
We should perhaps also remove Tom's email from:
```
src/misc/crypt/crypt.c:   "LibTomCrypt " SCRYPT " (Tom St Denis, tomstdenis@gmail.com)\n"
```

File: makefile.unix [hunk 1]
```diff
@@ -15,15 +15,20 @@
...
 #Or if you are using Intel C compiler you might need something like:
 #
 # make -f makefile.unix CC=icc AR=xiar CFLAGS="-fast -DUSE_LTM -DLTM_DESC -I/path/to/libtommath" EXTRALIBS=/path/to/libtommath/libtommath.a all
 #
 
 #The following can be overridden from command line e.g. "make -f makefile.unix CC=gcc ARFLAGS=rcs"
-DESTDIR   = /usr/local
+DESTDIR   =
+PREFIX    = /usr/local
+LIBPATH   = $(DESTDIR)$(PREFIX)/lib
```
Comment (karel-m) [inline makefile.unix]: You probably wanted to use `LIBPATH` & co. later in the `makefile.unix` or not?

Comment (sjaeckel) [inline makefile.unix]: Ah, I forgot to remove that again... or do you think we should use `LIBPATH` & co. in all makefiles?

Comment (karel-m) [inline makefile.unix]: I am fine with having just PREFIX in makefile.unix

File: makefile.unix [hunk 2]
```diff
@@ -15,15 +15,16 @@
...
+# make -f makefile.unix PREFIX=/opt/libtom install
 #
 #Or if you are using Intel C compiler you might need something like:
 #
 # make -f makefile.unix CC=icc AR=xiar CFLAGS="-fast -DUSE_LTM -DLTM_DESC -I/path/to/libtommath" EXTRALIBS=/path/to/libtommath/libtommath.a all
 #
 
 #The following can be overridden from command line e.g. "make -f makefile.unix CC=gcc ARFLAGS=rcs"
-DESTDIR   = /usr/local
+DESTDIR   =
```
Comment (karel-m) [inline makefile.unix]: The `DESTDIR   =` should be perhaps completely removed

Comment (sjaeckel) [inline makefile.unix]: Ah damn, yeah... or we use it so #232 would also be solved for makefile.unix

File: helper.pl [hunk 3]
```diff
@@ -136,6 +136,8 @@ sub check_comments {
 MARKER
   my @all_files;
   find({ wanted=> sub { push @all_files, $_ if $_ =~ /\.(c|h)$/ }, no_chdir=>1 }, 'src');
+  find({ wanted=> sub { push @all_files, $_ if $_ =~ /\.(c|h)$/ }, no_chdir=>1 }, 'demos');
```
Comment (karel-m) [inline helper.pl]: better:
```perl
find({ wanted=> sub { push @all_files, $_ if $_ =~ /\.(c|h)$/ }, no_chdir=>1 }, 'src', 'demos', 'tests');
```

**Comment by karel-m** [comment]
👍 I am for merging (just small cosmetics in helper.pl - see my comment)

File: doc/crypt.tex [hunk 4]
```diff
@@ -51,13 +51,13 @@
...
 	\section{#1}
-   \markboth{\textsf{www.libtom.org}}{\thesection ~ {#1}}
+   \markboth{\textsf{www.libtom.net}}{\thesection ~ {#1}}
 	}
 
 \newcommand{\mystarsection}[1]    % Re-define the chaptering command to use
 	{                   % THESE headers.
 	\section*{#1}
-   \markboth{\textsf{www.libtom.org}}{{#1}}
+   \markboth{\textsf{www.libtom.net}}{{#1}}
```
Comment (sjaeckel) [inline doc/crypt.tex]: Good like that?

**Pull request merged**